### PR TITLE
Use of Managed AppRouter (instead of Standalone AppRouter)

### PR DESCRIPTION
**Pull request by raz-korn**
The change affects several tutorials in this tutorial group.
In addition the "https://developers.sap.com/tutorials/cp-portal-cloud-foundry-getting-started.html" tutorial should be added to the group. It should be just before the "Create a Dev Space" tutorial.

### CODE REVIEW

File: tutorials/appstudio-devspace-fiori-create/appstudio-devspace-fiori-create.md [hunk 1]
```diff
@@ -64,7 +66,9 @@ Dev spaces are like isolated virtual machines in the cloud that can be instantly
 
 The SAP Fiori dev space opens and the **Welcome** tab appears.
 
-!![Open Dev Space](03-01-02 AppStudio Welcome Tab.jpg)
+!![Open Dev Space](AppStudio Welcome Tab_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-devspace-fiori-create/appstudio-devspace-fiori-create.md:69]: Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 2]
```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
 
 [ACCORDION-BEGIN [Step 1: ](Create new SAP Fiori project)]
 
-1. In the **Welcome Page** click **Create project from a template**.
+1. In the **Welcome Page** click **Start from template**.
 
-    !![Open Dev Space](03-01-02 AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
+    !![Open Dev Space](AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:33]: Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 3]
```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
...
 
-    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **SAP Business Application Studio: Welcome Page**. The command palette is opened at the top-center of the SAP Business Application Studio window.
+    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **Welcome**. The command palette is opened at the top-center of the SAP Business Application Studio window.
 
-    >!![Welcome Page from command palette](03-01-03 AppStudio Welcome Page from Command Palette_.jpg)
+    >!![Welcome Page from command palette](AppStudio Welcome Page from Command Palette_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:37]: Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 4]
```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
...
     > You can also create a project from the terminal using Yeoman.
 
-2. Make sure that the target folder is `/home/user/projects`, select the **SAP Fiori Freestyle Project** template, and click **Next**.
+2. For convenience, click the **Explorer** view button to close the `Explorer` view.
 
-    !![Fiori project template](03-03 AppStudio Fiori Project Template_.jpg)
+3. Make sure that the target folder is `/home/user/projects`, select the **SAP Fiori Freestyle Project** template, and click **Start**.
 
-3. For **Target Running Environment**, select the following, and click **Next**.
+    !![Fiori project template](AppStudio Fiori Project Template_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:49]: Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 5]
```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
...
+
+4. For **Target Running Environment**, select the following, and click **Next**.
 
     | Step | Parameter | Value |
     |:-----|:----------|:------|
     | A | Select the target running environment | **Cloud Foundry** |
     | B | Select the template you want to use | **SAPUI5 Application** |
 
-    !![fiori project template](03-04 AppStudio Fiori Project Template Target Running Environment_.png)
+    !![fiori project template](AppStudio Fiori Project Template Target Running Environment_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:58]: Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Hello Raz,
thank you for your PR. Please adjust all file names and file types of images according to the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64).

File: tutorials/appstudio-devspace-fiori-create/appstudio-devspace-fiori-create.md (hunk 1 above)
Comment (raz-korn) [reply tutorials/appstudio-devspace-fiori-create/appstudio-devspace-fiori-create.md:69]: done

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md (hunk 2 above)
Comment (raz-korn) [reply tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md]: done

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md (hunk 3 above)
Comment (raz-korn) [reply tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md]: done

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md (hunk 4 above)
Comment (raz-korn) [reply tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md]: done

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md (hunk 5 above)
Comment (raz-korn) [reply tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md]: done

**Comment by raz-korn** [comment]
Hi @MichaelCzcz ,

Per your request - changes for file names and types are done.

Regards - Raz

**Pull request merged**